CHANGELOG.md

```diff
@@ -19,5 +19,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add `ResolveVersion`: resolves version of a reference.
 - Add `TopLevel`: finds absolute path of top-level git directory.
 
-[Unreleased]: https://github.com/giantswarm/architect-orb/compare/v0.1.0...HEAD
+## [0.1.0] - 2019-10-10
```

Comment:
Can you also push the tag for the old one?

Reply:
Yes sure.